config/config.ini.example

```diff
@@ -18,7 +18,7 @@
 #geofence:										# File containing geofence(s) used to filter (default: None)
 #location:										# Location for the manager. 'Name' or 'lat lng' (default: None)
 #locale:										# Language to be used to translate names (default: en)
-#cache_type:                                    # Type of cache to use. (default: 'mem')
+#cache_type:                                    # Type of cache to use: mem or file. (default: 'mem')
```

Comment:
Change this to `Specify the type of cache to use. Options: ['mem', 'file'] (Default: 'mem')`